### ISSUE TITLE
Script hangs when using 'list' type prompt

### ISSUE THREAD

**Issue by jbadlato** [SBoudrias/Inquirer.js#807]
When running this script:
```
const inquirer = require('inquirer');

inquirer
  .prompt([
	{
		type: 'list',
		name: 'reptile',
		message: 'which is better?',
		choices: ['alligator','crocodile'],
	}
  ])
  .then(answers => {
    console.info('Answers:', answers);
  });
```
It outputs the answers to the console, as expected.  However, the script hangs and does not end.  I've tested this in Powershell and git bash on Windows with the same results.

**Comment by oailloud** [comment]
Same here on a bash MacOS.
I downgraded to the v6.3.1 where it works.

**Comment by krohrsb** [comment]
Same here bash/fish on MacOS. v6.3.1 works. Note this also causes other plugins like 'fuzzypath' to hang.

**Comment by jbadlato** [comment]
Looks like this issue has been fixed in 6.4.1.
